src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -189,9 +189,12 @@ public bool UsingDefaultValue
             get { return usingDefaultValue; }
             set
             {
-                usingDefaultValue = value;
-                RaisePropertyChanged("UsingDefaultValue");
-                RaisePropertyChanged("ToolTip");
+                if (usingDefaultValue != value)
+                {
+                    usingDefaultValue = value;
+                    RaisePropertyChanged("UsingDefaultValue");
+                    RaisePropertyChanged("ToolTip");
```

Comment:
I like the fix here. Also curious does similar problem not exist for other properties, e.g. X, Y etc?

Reply:
Yes, this pattern is used in most places. For `X` and `Y`, I noticed they are deprecated and there is a `Position` which is a way to update both. Maybe that's the reason why it's different.